Speedykom/Regional-Pandemic-Analytics
# fix: airflow openid session + add logout

In this PR : 
- Clean up and align airflow keycloak Oauth config
- Check for active sessions in keycloak and logout when there's no sessions (for superset and airflow)

How to test: 
- Login in superset and airflow
- Connect to keycloak as admin
- Delete the user session
- It should log you out automatically from superset and airflow whithin 10 sec
